engines/python/setup/djl_python/transformers_neuronx.py

```diff
@@ -189,6 +188,8 @@ def parse_input(self, inputs):
 
                 if not "output_formatter" in param:
                     param["output_formatter"] = self.config.output_formatter
+                if self.rolling_batch:
+                    param["stream"] = input_map.pop("stream", False)
```

Comment:
Also needed for trtllm

Reply:
trtllm is only rolling batch right now, but i know @sindhuvahinis is adding support for dynamic batch there. 

Sindhu, should i add the non rolling batch condition in trtllm as well? Not sure how the changes you will be raising will look, so maybe you can add it in your PR as well?

Reply:
Yeah will add it in my PR.